src/standard/configure.html

```diff
@@ -45,11 +45,13 @@
 
     // storage for configuration
     _setupConfigure: function(initialConfig) {
-      this._config = {};
+      // If we have more non-undefined properties, it is more efficient to clone whole object
+      // and then remove undesired properties
+      this._config = Object.create(initialConfig || {});
```

Comment:
I think all we need here is:

`this._config = initialConfig ? Object.create(initialConfig) : {};`

doesn't seem like the delete is necessary